client/src/components/PizzaPlaceShow.js

```diff
@@ -11,6 +20,7 @@ const PizzaPlaceShow = props => {
 				throw new Error(`${response.status} (${response.statusText})`)
 			}
 			const pizzaData = await response.json()
+			debugger
```

Comment:
big big no. avoidable issues like this will hurt you in terms of getting the special reward for clean PRs!